aten/src/ATen/native/Distributions.cpp

```diff
@@ -274,7 +274,7 @@ Tensor& normal_out(const Tensor& mean, double std, c10::optional<Generator> gen,
 
 Tensor& normal_out_meta(const Tensor& mean, double std, c10::optional<Generator> gen, Tensor& output) {
   CHECK_NORMAL_STD(std);
-  auto std_tensor = at::empty_like(output, MemoryFormat::Contiguous);
+  auto std_tensor = at::empty_like(output, c10::kMeta, MemoryFormat::Contiguous);
```

Comment:
Ok, this is the fix for the comment from the previous PR (https://github.com/pytorch/pytorch/pull/70089/files#r773235114). Next time, it would be a bit easier to keep track of if it was bundled into the same PR :)

Reply:
@bdhirsh yeah, sorry, i'll squash and let you know